numpy/__init__.pyi

```diff
@@ -44,6 +44,28 @@ if sys.version_info >= (3, 8):
 else:
     from typing_extensions import Literal, Protocol
 
+# Ensures that the stubs are picked up
+from . import (
+    char,
+    compat,
+    core,
+    ctypeslib,
+    emath,
+    fft,
+    lib,
+    linalg,
+    ma,
+    math,
+    matrixlib,
+    os,
+    polynomial,
+    random,
+    rec,
+    testing,
+    version,
+    warnings,
```

Comment:
You should be ignoring all the ones that are not part of the numpy installation
```suggestion
    char,
    compat,
    core,
    ctypeslib,
    emath,
    fft,
    lib,
    linalg,
    ma,
    matrixlib,
    polynomial,
    random,
    rec,
    testing,
    version,
```

Reply:
Removed the last ones as of https://github.com/numpy/numpy/pull/17104/commits/160a782381d2f723b0f1d836503e13eedeaaf7aa.